test/raven.utils.js

```diff
@@ -108,6 +108,23 @@ describe('raven.utils', function() {
       parseStack([], callback);
       parseStack([{lol: 1}], callback);
     });
+
+    it('should extract context from last stack line', function(done){
```

Comment:
But to touch the branch that you added, that mandates that this test is run on Windows, correct? If the test is run on anything non-windows, your code branches never get touched. So I agree the test is useful, we just need to mock it to force a windows style frame.filename so that we can assert it does the right thing. Especially since all testing is run in Linux or OSX.